packages/web/src/pages/sign-up-page/components/HandleField.tsx

```diff
@@ -39,18 +48,53 @@ type HandleFieldProps = Partial<HarmonyTextFieldProps> & {
   onErrorSocialMediaLogin?: () => void
 }
 
+/**
+ * This is a workaround because Formik's isValidating only returns true DURING the validation.
+ * However for this scenario we want to wait for the results of the validation before showing any messaging.
+ * To do this we wait for Formik isValidating to change to true, then wait for it to change
+ * back to false before saying we're "not waiting" for validation anymore
+ * @returns {isWaitingOnValidation, setIsWaitingOnValidation}
+ */
+const useIsWaitingOnValidation = () => {
+  const { isValidating } = useFormikContext()
+  const [isWaitingOnValidation, setIsWaitingOnValidation] = useState(false)
+  const [previousIsValidating, setPreviousIsValidating] = useState(false)
```

Comment:
usePrevious check out I think we use that somewhere in sign up for the same reason